toybox/Dockerfile

```diff
@@ -16,13 +16,15 @@ WORKDIR /toybox
 
 # https://landley.net/toybox/downloads/?C=M;O=D
 # https://github.com/landley/toybox/releases
-ENV TOYBOX_VERSION 0.8.5
+ENV TOYBOX_VERSION 0.8.6
 
 RUN set -eux; \
 	wget -O toybox.tgz "https://landley.net/toybox/downloads/toybox-$TOYBOX_VERSION.tar.gz"; \
 	tar -xf toybox.tgz --strip-components=1; \
 	rm toybox.tgz
 
+# Workaround for bug in scripts/mkroot.sh that exits 1 when BUILTIN=1
+RUN sed -i '273,275d' scripts/mkroot.sh
```

Comment:
Now that we've got https://github.com/landley/toybox/commit/ea4748a7cbfa5e2f3ef188f917d4e5aeac70dd0f, I'd rather apply that patch (esp. instead of blindly deleting specific line numbers :sweat_smile: :see_no_evil:), maybe something like this:

```suggestion
# workaround for bug in scripts/mkroot.sh that exits 1 when BUILTIN=1; https://github.com/landley/toybox/issues/305
RUN set -eux; \
	wget -O builtin-fix.patch 'https://github.com/landley/toybox/commit/ea4748a7cbfa5e2f3ef188f917d4e5aeac70dd0f.patch'; \
	patch -p1 --input=builtin-fix.patch; \
	rm builtin-fix.patch

```

(This would also mean we need to add `patch` to the list of dependencies installed above.)

I'm happy to take over from here if you'd rather not continue sinking more time into this -- it's appreciated either way! :smile: :+1:

Reply:
No worries - it should be all set - not sure whether to add a note in there to remove in 0.8.7.  Do you want me to squash into one commit?

Reply:
Eh, it'll probably fail to apply, so should be pretty obvious 😅

I'll just squash when I merge (after CI passes); thanks! 👍